internal/hcs/schema2/cimfs.go

```diff
@@ -10,11 +10,20 @@
 package hcsschema
 
 const (
-	CimMountFlagNone         uint32 = 0x0
-	CimMountFlagChildOnly    uint32 = 0x1
-	CimMountFlagEnableDax    uint32 = 0x2
-	CimMountFlagCacheFiles   uint32 = 0x4
-	CimMountFlagCacheRegions uint32 = 0x8
+	CimMountFlagNone       uint32 = 0x0
+	CimMountFlagEnableDax  uint32 = 0x2
+	CimMountBlockDeviceCim uint32 = 0x10
+	CimMountSingleFileCim  uint32 = 0x20
+
+	CimCreateFlagNone                uint32 = 0x0
+	CimCreateFlagDoNotExpandPEImages uint32 = 0x1
+	CimCreateFlagFixedSizeChunks     uint32 = 0x2
+	CimCreateFlagBlockDeviceCim      uint32 = 0x4
+	CimCreateFlagSingleFileCim       uint32 = 0x8
+
+	CimMergeFlagNone        uint32 = 0x0
+	CimMergeFlagSingleFile  uint32 = 0x1
+	CimMergeFlagBlockDevice uint32 = 0x2
```

Comment:
Are these actually part of HCS schema? If not, they should be in another package.

Reply:
Just export them from pkg/cimfs?